app/packages/core/src/components/Filters/NumericFieldFilter.tsx

```diff
@@ -238,9 +251,7 @@ const NumericFieldFilter = ({
           />
         ) : null}
         {(hasDefaultRange ||
-          nonfinites.some(
-            ([_, { value: v }]) => v !== nonfinites[0][1].value
-          ) ||
+          (hasNone && isSliderAtInitialPostion) ||
```

Comment:
If `isSliderAtInitialPosition` and `hasDefaultRange` have the same definition then this could become:
```js
        {(hasNone && hasDefaultRange) ||
```